community/patch/build

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+export CC="gcc -static"
+
+./configure \
+    --prefix=/usr \
+    --program-prefix=g
+
+make DESTDIR="$1" install
```

Comment:
Missing standalone `make`?

Reply:
i did not know we preferred that. will do

Reply:
I did notice once in a blue moon a package will fail without the standalone make. *cough* iptables

Reply:
done